lib/benchee/system.ex

```diff
@@ -108,29 +109,29 @@ defmodule Benchee.System do
     parse_memory_for(:Linux, system_cmd("cat", ["/proc/meminfo"]))
   end
 
-  @kilobyte_to_gigabyte 1024 * 1024
-  @byte_to_gigabyte 1024 * @kilobyte_to_gigabyte
-
   defp parse_memory_for(_, "N/A"), do: "N/A"
   defp parse_memory_for(:Windows, raw_output) do
     [memory] = Regex.run(~r/\d+/, raw_output)
     {memory, _} = Integer.parse(memory)
-    format_memory(memory, @byte_to_gigabyte)
+    format_memory(memory)
   end
   defp parse_memory_for(:macOS, raw_output) do
     {memory, _} = Integer.parse(raw_output)
-    format_memory(memory, @byte_to_gigabyte)
+    format_memory(memory)
   end
   defp parse_memory_for(:Linux, raw_output) do
     ["MemTotal:" <> memory] = Regex.run(~r/MemTotal.*kB/, raw_output)
     {memory, _} = memory
                   |> String.trim()
                   |> String.trim_trailing(" kB")
                   |> Integer.parse
-    format_memory(memory, @kilobyte_to_gigabyte)
+    format_memory(memory * 1024)
   end
 
-  defp format_memory(memory, coefficient), do: "#{Float.round(memory / coefficient, 2)} GB"
+  defp format_memory(memory) do
+    scaled_memory = Memory.scale(memory, :gigabyte)
+    Memory.format({scaled_memory, :gigabyte})
```

Comment:
I think just calling `Memory.format(memory)` should give us the right result, it mgiht not always be GB as before but if it decides not to do GB then I think that's what we want :)

Reply:
i did not want to change how it was implemented before... and i missed the point of implementing memory formatter :)